CONTRIBUTING.md

```diff
@@ -54,6 +54,13 @@ In the case of an objection being raised in a pull request by another collaborat
 * [`squash`][] pull-requests made up of multiple commits
 * Land how you like as long as there are no merge commits
 
+## How to become a collaborator
+
+* Collaborators must be actively contributing to the project
+* A collaborator issue must be opened on the @nodejs/nodejs.dev with user's github name in the title
+* Issue must be reviewed and approved by at least two members of @nodejs/nodejs.dev, @nodejs/website, or @nodejs/website-redesign 
+* After the issue is reviewed and approved it must stay for 48 hours without any objections
+* Once the requirements are met and approved, a member of @nodejs/community-committee will make the person an official collaborator
```

Comment:
What does "make the person an official collaborator" mean? If this request is done through a PR to add the person to the list, that is step one. I believe the only other step is to add them to the team that has access to this PR.

Reply:
it's done through the PR

Reply:
If its done through the PR, then what does this step mean? I think what is perhaps missing is adding them to the github team that manages this repo.

Reply:
```suggestion
* Once the requirements are met and approved, the pull request can be merged and a member of @nodejs/community-committee will add the new member to the @nodejs/nodejs-dev team
```